contracts/staking/validatorShare/IValidatorShare.sol

```diff
@@ -11,13 +11,18 @@ import {Lockable} from "../../common/mixin/Lockable.sol";
 import {StakingInfo} from "../StakingInfo.sol";
 import {IStakeManager} from "../stakeManager/IStakeManager.sol";
 
+
 contract IValidatorShare is ERC20, Lockable, Ownable {
     using SafeMath for uint256;
     StakingInfo public stakingLogger;
     IStakeManager public stakeManager;
     uint256 public validatorId;
     uint256 public validatorRewards;
     uint256 public commissionRate;
+    ///@notice # of checkpoints after which commission rate can be updated
```

Comment:
I think both `///` and `@notice` is meant for class, interface and functions.

Reply:
https://solidity.readthedocs.io/en/v0.5.10/natspec-format.html
Not sure what to use for variables. `if no tags are used then the Solidity compiler will interpret a /// or /** comment in the same way as if it were tagged with @notice.` so using notice makes sense.